client/app/config/steps/password.coffee

```diff
@@ -7,16 +7,23 @@ module.exports = {
 
     # If OK, return null
     # if not return an Array of errors
-    validate: (data) ->
-        return null
+    # that will be triggered throw onboarding
+    # to dispatch error into app
+    validate: (data={}) ->
+        unless data.password?
+            return { type: 'password', text:'step empty fields'}
+        else
+            return null
+
 
     save: (data) ->
-        data = JSON.stringify data
         return new Promise((resolve, reject) ->
             jQuery.post({
```

Comment:
As discussed before with @m4dz, and @gregorylegarec using fetch is an improvement but not the scope of this PR.